src/components/grid-view/GridViewWrapper.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import * as React from 'react';
+import GridView from './GridView';
+
+type Props = {
+    currentCollection: Collection,
+    height: number,
+    slotRenderer: (slotIndex: number) => React.Element<any>,
+    width: number,
+};
+
+const DEFAULT_COLUMN_COUNT = 5;
+const MAX_COLUMN_COUNT = 5;
+
+const ONE_COLUMN_BREAKPOINT = 700;
+const THREE_COLUMN_BREAKPOINT = 1400;
+
+const GridViewWrapper = ({ width, ...rest }: Props) => {
+    let maxColumns;
+    if (width < ONE_COLUMN_BREAKPOINT) {
+        maxColumns = 1;
+    } else if (width < THREE_COLUMN_BREAKPOINT) {
+        maxColumns = 3;
+    } else {
+        maxColumns = MAX_COLUMN_COUNT;
+    }
+
+    return <GridView columnCount={Math.min(DEFAULT_COLUMN_COUNT, maxColumns)} width={width} {...rest} />;
```

Comment:
Is the `Math.min` call or `DEFAULT_COLUMN_COUNT` needed? It looks like `maxColumns` is always a number and less than or equal to `DEFAULT_COLUMN_COUNT`.

Reply:
My thinking on this is that in the future, `DEFAULT_COLUMN_COUNT` will become a prop (named something like `columnCount`) that is passed in from `GridViewSlider`.  In that scenario, the `Math.min` call would be needed.